sci-libs/libmed/libmed-4.0.0-r1.ebuild

```diff
@@ -22,15 +24,15 @@ SRC_URI="http://files.salome-platform.org/Salome/other/${MY_P}.tar.gz"
 LICENSE="GPL-3 LGPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE="doc fortran mpi python static-libs test"
+IUSE="doc fortran mpi python test"
 
 REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 RDEPEND="
-	sci-libs/hdf5[fortran=,mpi=]
+	>=sci-libs/hdf5-1.10.0[fortran=,mpi=]
 	mpi? ( virtual/mpi[fortran=] )
 	python? ( ${PYTHON_DEPS} )
 "
-DEPEND="${RDEPEND}
```

Comment:
keep `DEPEND`

Reply:
This ebuild has been removed

Reply:
So I need DEPEND and BDEPEND then? Hmm... need to take a deeper look into the differences of DEPEND and BDEPEND I suppose

Reply:
I put DEPEND back in with my latest commit 68a2306